src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -528,6 +528,12 @@
   - name: Create a scope map that allows content/write and metadata/read actions for `hello-world` repository, and content/read action for `hello-world-again`.
     text: >
         az acr scope-map create -n MyScopeMap -r MyRegistry --repository hello-world content/write metadata/read --repository hello-world-again content/read --description "Sample scope map."
+  - name: Create a scope map that allows all repository actions for `test`, and all gateway actions for `connectedRegistry`.
```

Comment:
@rosanch Do we want to clarify here that these actions are for `connectedRegistry` in `Registry` mode?

Reply:
I thought about it. But I wasn't sure exactly what is the need for Gateway tokens and I'm not sure if there is another application the users might give to this commands.